### DISCUSSION TITLE
Export the attacking IP(s) to text file for blacklist

### DISCUSSION THREAD

**Discussion by wizhardhacker1** [telekom-security/tpotce#1512] [Q&A] [answered]
Anyone have any suggestions or has done a custom blacklist export from tpot? first time using this and would like the data to export the attacker ip in a text list and update every 15 min

**Comment by rxerium** [comment]
I'd be interested to know this too, did you find anything?

**Reply by t3chn0m4g3** [reply, marked as answer]
Like [this](https://github.com/telekom-security/tpotce/blob/master/docker/tpotinit/dist/bin/mytopips.sh)?

**Reply by rxerium** [reply]
Exactly like that thanks!
